web/src/stream_topic_history.ts

```diff
@@ -33,6 +33,23 @@ export function stream_has_topics(stream_id: number): boolean {
     return history.has_topics();
 }
 
+export function stream_has_named_topics(stream_id: number): boolean {
```

Comment:
`stream_has_locally_available_named_topics` would be a more correct name for this.